pkgs/development/python-modules/duckdb/default.nix

```diff
@@ -53,8 +62,14 @@ buildPythonPackage rec {
 
   preCheck = ''
     export HOME="$(mktemp -d)"
+    # duckdb directory prevents loading the python module
+    mv duckdb duckdb.pk
```

Comment:
can probably get rid of these two lines. the `--inplace` added to `setupPyBuildFlags` may be the proper fix for this.

Reply:
Done, thanks